config/phpcs/composer.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "varying-vagrant-vagrants/vvv",
+  "type": "metapackage",
+  "description": "An open source Vagrant configuration for developing with WordPress",
+  "keywords": [
+    "vagrant"
```

Comment:
maybe remove this in favour of phpcs?

Reply:
This goes back to the `vvv/phpcs` suggestion. I'd keep `vagrant`, but add `phpcs` as well.